ArcadiaParties.Data/Helpers/AutoMapperProfiles.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Models;
+using AutoMapper;
+
+namespace ArcadiaParties.Data.Helpers
+{
+    class AutoMapperProfiles : Profile
+    {
+        public AutoMapperProfiles()
```

Comment:
Where is it used?

Reply:
Mapper injected as service and is used by UserRepository to transfer DTO to view layer.

Reply:
I know what mapper itself is. I asked, where and why is this profile class used.

Reply:
I've changed mapper injection